### ISSUE TITLE
cannot delete `=` when let type annotation trails with a hole

### ISSUE THREAD

**Issue by cyrus-** [hazelgrove/hazel#1295]
<img width="611" alt="image" src="https://github.com/hazelgrove/hazel/assets/280638/fdbd7d47-e3d1-4914-912c-6979767f86fb">

**Comment by disconcision** [comment]
I don't believe there is any situation in which one can delete the `=` while the `let `and `in` are both down; see #1088

**Comment by disconcision** [comment]
Dupe of #1088.
